msgcntr/messageforums-app/src/webapp/jsp/discussionForum/statistics/dfStatisticsListByAllTopics.jsp

```diff
@@ -38,8 +38,9 @@
           <h:panelGroup styleClass="itemNav specialLink">        
           	<h:commandLink action="#{ForumTool.processActionStatistics}" value="#{msgs.cdfm_statistics} #{msgs.stat_byUser}" title="#{msgs.cdfm_statistics} #{msgs.stat_byUser}"/>
 			<h:outputText value=" #{msgs.cdfm_toolbar_separator} " />
-			<h:outputText value="#{msgs.cdfm_statistics} #{msgs.stat_byTopic}"/>
-		  </h:panelGroup>
+			<h:outputText value="#{msgs.cdfm_statistics} #{msgs.stat_byTopic}" style="padding-right: 5px;"/>
```

Comment:
Thanks for removing the nbsp. 

Another case where a class using the $standard-space variable could be reused for the spacing.